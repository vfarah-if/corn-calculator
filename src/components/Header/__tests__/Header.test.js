import React from "react";
import { render } from "@testing-library/react";

import {
  DefaultHeader,
} from "../Header.stories";

describe("Header", () => {
  test("should create a header with custom image url", () => {
    const { container } = render(<DefaultHeader {...DefaultHeader.args} />);

    expect(container).toMatchSnapshot("default");
  });
});