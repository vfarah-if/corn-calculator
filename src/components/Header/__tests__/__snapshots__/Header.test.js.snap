// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should create a header with custom image url: default 1`] = `
<div>
  <header
    class="header"
    style="background-image: url(https://images.pexels.com/photos/34153/pexels-photo.jpg?auto=compress&cs=tinysrgb&h=350); background-position: center; background-size: cover; background-repeat: no-repeat;"
  >
    <h1>
      Title to add to the top of a component
    </h1>
  </header>
</div>
`;
